ship.config.js

```diff
@@ -1,35 +1,49 @@
 /* eslint-disable import/no-commonjs */
 
-const fs = require('fs');
-const path = require('path');
+const packages = [
+  'packages/autocomplete-core',
+  'packages/autocomplete-preset-algolia',
+  'packages/autocomplete-react',
+];
 
 module.exports = {
+  monorepo: {
+    mainVersionFile: 'lerna.json',
+    packagesToBump: packages,
+    packagesToPublish: packages,
+  },
   mergeStrategy: {
     toSameBranch: ['next'],
   },
   publishCommand({ tag }) {
     return `yarn publish --access public --tag ${tag}`;
   },
-  versionUpdated({ version, dir }) {
-    // Bump the string version in the version file
-    const versionPath = path.resolve(dir, 'src/version.ts');
-    fs.writeFileSync(versionPath, `export const version = '${version}';\n`);
-  },
-  // @TODO: this will be possible once this is addressed:
-  // https://github.com/algolia/shipjs/issues/604
-  // afterPublish({ exec, dir }) {
-  //   // Update the Autocomplete.js version in the examples
-  //   const examplePath = path.resolve(dir, 'examples/autocomplete.js');
-  //   const { version } = require('./package.json');
+  versionUpdated({ version, exec }) {
+    const update = ({ package, deps }) => {
+      /*
+        With caret(^), the dependencies need to be quoted, like the following:
+        > yarn workspace abc add "def@^1.0.0" "ghi@^1.0.0"
+      */
+      const depsWithVersion = deps.map(dep => `"${dep}@^${version}"`).join(' ');
+      exec(`yarn workspace ${package} add ${depsWithVersion}`);
```

Comment:
We probably don't want to update the `dependencies` in the `package.json` of the packages because they need to be loose. We should do that manually when needed.

Reply:
Does that mean we're going to have independent versioning across the packages?

Reply:
So, even if we release new versions of `autocomplete-core` or `autocomplete-preset-algolia`, you want to validate if we want to release `autocomplete-react` to include the new versions of the two?

`autocomplete-core` has no dependencies.
`autocomplete-preset-algolia` has no dependencies.
`autocomplete-react` depends on the two.

Let's say,

`autocomplete-core`: `1.0.0` -> `1.1.0`
`autocomplete-preset-algolia`: `1.0.0` -> `1.1.0`
`autocomplete-react`: `1.0.0` -> stays the same

could you help me understand in what kind of situation we'd want to do it?

Reply:
As discussed IRL, looks good actually!